src/schema/v2/artwork/updateArtworkMutation.ts

```diff
@@ -0,0 +1,49 @@
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import Artwork from "schema/v2/artwork"
+import { omitBy } from "lodash"
+
+interface UpdateArtworkMutationInputProps {
+  id: string
+  availability?: boolean
+}
+
+export const updateArtworkMutation = mutationWithClientMutationId<
```

Comment:
I'm a little confused what this mutation has to do with CMS conversations?

Reply:
I've been following this guide that @lidimayra put together: https://www.notion.so/artsy/Backend-endpoints-needed-for-conversations-revamp-29857322665b48e488b308002305704d

There are reasons in there 👍